bgpd/bgp_fsm.c

```diff
@@ -491,11 +491,14 @@ static void bgp_connect_timer(struct event *thread)
 	assert(!connection->t_read);
 
 	if (bgp_debug_neighbor_events(peer))
-		zlog_debug("%s [FSM] Timer (connect timer expire)", peer->host);
+		zlog_debug("%s [FSM] Timer (connect timer (%us) expire)", peer->host,
+			   peer->v_connect);
 
 	if (CHECK_FLAG(peer->sflags, PEER_STATUS_ACCEPT_PEER))
 		bgp_stop(connection);
 	else {
+		if (!peer->connect)
```

Comment:
why do we need to test for peer->connect here?

Reply:
This is to check if we have "neighbor X timers connect N" set or not. If set - do not use backoff. Or do you think we should do it also?